### DISCUSSION TITLE
GitLab Support

### DISCUSSION THREAD

**Discussion by niclasheinz** [giscus/giscus#1581] [Ideas]
Hello together

Discus is a very cool project and ideal for static web hosting. However, I would also like to host the comments under GitLab. One possibility in my eyes is that we work via issues and simply add the label ‘discus’ or similar to the issue.

**Comment by unode** [comment]
GitLab has a [Discussions API](https://docs.gitlab.com/api/discussions/) that could help implementing this feature.

**Comment by Sieboldianus** [comment]
I find Giscus really awesome - it is a perfect solution for static sites. I would find it even more awesome if it would work with a self-hosted Gitlab.

**Reply by Sieboldianus** [reply]
Note: There's Beblob [1], Demo [2]. It doesn't yet seem to work with not-logged in anonymous display of posts, which makes it impractical for a blog.


[1]: https://gitlab.com/antonbelev/beblob
[2]: https://belev.me/2025/02/20/Introducing-BeBlob-A-GitLab-Powered-Comment-Section-for-Static-Websites/
